coordinator/src/services/translation/cancellation/cancellation-response.ts

```diff
@@ -44,47 +44,47 @@ function convertResourceToBundleEntry(resource: fhir.Resource) {
 function createBundleEntries(cancellationResponse: CancellationResponse) {
   const fhirPatient = createPatient(cancellationResponse.recordTarget.Patient)
 
-  const hl7ResponsiblePartyAgentPerson = cancellationResponse.responsibleParty.AgentPerson
   const hl7AuthorAgentPerson = cancellationResponse.author.AgentPerson
+  const hl7ResponsiblePartyAgentPerson = cancellationResponse.responsibleParty?.AgentPerson
 
   const {
-    fhirPractitioner: fhirResponsiblePartyPractitioner,
-    fhirLocations: fhirResponsiblePartyLocations,
-    fhirHealthcareService: fhirResponsiblePartyHealthcareService,
-    fhirPractitionerRole: fhirResponsiblePartyPractitionerRole
-  } = convertAgentPerson(hl7ResponsiblePartyAgentPerson)
+    fhirPractitioner: fhirCancelRequesterPractitioner,
+    fhirLocations: fhirCancelRequesterLocations,
+    fhirHealthcareService: fhirCancelRequesterHealthcareService,
+    fhirPractitionerRole: fhirCancelRequesterPractitionerRole
+  } = convertAgentPerson(hl7AuthorAgentPerson)
 
   const unorderedBundleResources: Array<fhir.Resource> = [
     fhirPatient,
-    fhirResponsiblePartyPractitioner,
-    ...fhirResponsiblePartyLocations,
-    fhirResponsiblePartyHealthcareService,
-    fhirResponsiblePartyPractitionerRole
+    fhirCancelRequesterPractitioner,
+    ...fhirCancelRequesterLocations,
+    fhirCancelRequesterHealthcareService,
+    fhirCancelRequesterPractitionerRole
   ]
 
-  let requesterId = fhirResponsiblePartyPractitioner.id
+  let requesterId = fhirCancelRequesterPractitioner.id
```

Comment:
maybe rename to authorId? (sorry)

Reply:
yea i was rereading the code and requester is pretty confusing here - its requesterId at the moment because its used to populate the `requester` field on a MedicationRequest